app/cart.js

```diff
@@ -1,24 +1,48 @@
 //import data and scripts
-
```

Comment:
to get into the habit of writing clean code, make sure to keep all imports at the top with no extra spaces.

Reply:
Thanks, I've fixed it.
Looking at it I think my reasoning was to separate the data imports from the function imports, is that not necessary?
Or should I do it like this:

//import static data
import tapes from './tapes.js';

//import functions
import { renderLineItem } from './render-line-item.js';